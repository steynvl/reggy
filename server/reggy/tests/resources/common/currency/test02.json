{
  "type": "Currency",
  "information": {
    "currencies": [
      "Afghan afghani",
      "European euro",
      "Algerian dinar",
      "East Caribbean dollar",
      "Bahraini dinar",
      "Australian dollar",
      "United States dollar",
      "Belize dollar",
      "Zambian kwacha",
      "Vietnamese dong",
      "Uzbekistani som",
      "Ukrainian hryvnia",
      "Turkmen manat"
    ]
  },
  "generalRegexInfo": {
    "startRegexMatchAt": "Start of word",
    "endRegexMatchAt": "End of word",
    "regexTarget": "Java"
  },
  "generateMethod": "commonUseCases"
}
