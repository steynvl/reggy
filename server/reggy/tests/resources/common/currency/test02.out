Java
Pattern regex = Pattern.compile("\\b(?:AFN|EUR|DZD|XCD|BHD|AUD|USD|BZD|ZMW|VND|UZS|UAH|TMT)\\b");
\\b(?:AFN|EUR|DZD|XCD|BHD|AUD|USD|BZD|ZMW|VND|UZS|UAH|TMT)\\b
Perl
my $regex = /\b(?:AFN|EUR|DZD|XCD|BHD|AUD|USD|BZD|ZMW|VND|UZS|UAH|TMT)\b/;
\b(?:AFN|EUR|DZD|XCD|BHD|AUD|USD|BZD|ZMW|VND|UZS|UAH|TMT)\b
POSIX
\b(?:AFN|EUR|DZD|XCD|BHD|AUD|USD|BZD|ZMW|VND|UZS|UAH|TMT)\b
\b(?:AFN|EUR|DZD|XCD|BHD|AUD|USD|BZD|ZMW|VND|UZS|UAH|TMT)\b
Python
regex = re.compile(r'\b(?:AFN|EUR|DZD|XCD|BHD|AUD|USD|BZD|ZMW|VND|UZS|UAH|TMT)\b')
\b(?:AFN|EUR|DZD|XCD|BHD|AUD|USD|BZD|ZMW|VND|UZS|UAH|TMT)\b
JavaScript
const regex = /\b(?:AFN|EUR|DZD|XCD|BHD|AUD|USD|BZD|ZMW|VND|UZS|UAH|TMT)\b/;
\b(?:AFN|EUR|DZD|XCD|BHD|AUD|USD|BZD|ZMW|VND|UZS|UAH|TMT)\b
PHP
$regex = '/\b(?:AFN|EUR|DZD|XCD|BHD|AUD|USD|BZD|ZMW|VND|UZS|UAH|TMT)\b/';
\b(?:AFN|EUR|DZD|XCD|BHD|AUD|USD|BZD|ZMW|VND|UZS|UAH|TMT)\b
Golang
regex, _ := regexp.Compile("\\b(?:AFN|EUR|DZD|XCD|BHD|AUD|USD|BZD|ZMW|VND|UZS|UAH|TMT)\\b")
\\b(?:AFN|EUR|DZD|XCD|BHD|AUD|USD|BZD|ZMW|VND|UZS|UAH|TMT)\\b
Rust
let re = Regex::new(r"\b(?:AFN|EUR|DZD|XCD|BHD|AUD|USD|BZD|ZMW|VND|UZS|UAH|TMT)\b").unwrap();
\b(?:AFN|EUR|DZD|XCD|BHD|AUD|USD|BZD|ZMW|VND|UZS|UAH|TMT)\b
C#
var re = new Regex(@"\b(?:AFN|EUR|DZD|XCD|BHD|AUD|USD|BZD|ZMW|VND|UZS|UAH|TMT)\b");
\b(?:AFN|EUR|DZD|XCD|BHD|AUD|USD|BZD|ZMW|VND|UZS|UAH|TMT)\b
Scala
val re = "\\b(?:AFN|EUR|DZD|XCD|BHD|AUD|USD|BZD|ZMW|VND|UZS|UAH|TMT)\\b".r
\\b(?:AFN|EUR|DZD|XCD|BHD|AUD|USD|BZD|ZMW|VND|UZS|UAH|TMT)\\b
Kotlin
val regex = Regex("\\b(?:AFN|EUR|DZD|XCD|BHD|AUD|USD|BZD|ZMW|VND|UZS|UAH|TMT)\\b")
\\b(?:AFN|EUR|DZD|XCD|BHD|AUD|USD|BZD|ZMW|VND|UZS|UAH|TMT)\\b